### PR TITLE
Editor performance improvement

### PR DESCRIPTION
**Pull request by nubsthead**

### CODE REVIEW

File: src/view/event-panel-edit/editor-small-view.jsx [hunk 1]
```diff
@@ -70,6 +64,11 @@ export default function EditorSmallView({
 		setDropZoneEnable(true);
 	};
 
+	useEffect(() => {
+		console.clear();
+		console.log(data);
+	}, [data]);
+
```
Comment (giuliano176) [inline src/view/event-panel-edit/editor-small-view.jsx]: is this console.log intentional?

File: src/view/event-panel-edit/editor-small-view.jsx [hunk 2]
```diff
@@ -70,6 +64,11 @@ export default function EditorSmallView({
 		setDropZoneEnable(true);
 	};
 
+	useEffect(() => {
```
Comment (zovomat) [inline src/view/event-panel-edit/editor-small-view.jsx]: Remove the logs please :smile:

**Review by zovomat — DISMISSED** [review]

File: src/view/event-panel-edit/editor-small-view.jsx (hunk 1 above)
Comment (nubsthead) [reply src/view/event-panel-edit/editor-small-view.jsx]: It wasn't and it has been fixed now, please re-check if everything is ok (only thatuseeffect has been deleted in last commit)

Comment (zovomat) [reply src/view/event-panel-edit/editor-small-view.jsx]: resolving

**Review by zovomat — APPROVED** [review]

**Review by giuliano176 — APPROVED** [review]

**Pull request merged**